AuthenticatorShared/UI/Platform/Tabs/TabCoordinator.swift

```diff
@@ -103,5 +103,8 @@ final class TabCoordinator: Coordinator, HasTabNavigator {
             .settings(.settings): settingsNavigator,
         ]
         tabNavigator.setNavigators(tabsAndNavigators)
+
+        settingsNavigator.tabBarItem.accessibilityIdentifier = TabRoute.settings(.settings).accessibilityIdentifier
+        itemListNavigator.tabBarItem.accessibilityIdentifier = TabRoute.itemList(.list).accessibilityIdentifier
```

Comment:
🎨 If you want to pull the identifier out to a property, you could apply it programmatically here, instead of manually:
```
tabsAndNavigators.forEach { (key, value) in
    (value as? UINavigationController)?.tabBarItem.accessibilityIdentifier = key.accessibilityIdentifier
}
```

Reply:
thanks for the suggestion @KatherineInCode ! I made the change, validated the IDs appear using Appium Inspector.